src/main/build/debug.h

```diff
@@ -105,6 +105,7 @@ typedef enum {
     DEBUG_ATTITUDE,
     DEBUG_VTX_MSP,
     DEBUG_GPS_DOP,
+    DEBUG_ANGLE_MODE,
```

Comment:
Suggest `ANGLEMODE` or `FEEDFORWARD_ANGLE` as this is more consistent with existing debug parameters.
The second does fit more with the cli commands.

Reply:
ANGLEMODE is fine. Renamed.